lib/order.rb

```diff
@@ -0,0 +1,53 @@
+class Order
+
+  attr_reader :order_as_hash, :menu, :actual_total, :customer_supplied_total, :text
+
+  def initialize(menu = Menu.new, text = Text.new)
+    @menu = menu
+    @text = text
+  end
+
+  def place_order(dishes)
+    get_customer_total(dishes)
+    check_order(dishes)
+    text_confirmation(text)
+  end
+
+  def check_order (dishes)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)